DigitalLearningSolutions.Data.Tests/Services/CourseServiceTests.cs

```diff
@@ -244,13 +266,31 @@ public void UpdateLearningPathwayDefaultsForCourse_calls_data_service()
                 .MustHaveHappened();
         }
 
+        [Test]
+        public void RemoveDelegateFromCourse_returns_false_if_no_current_progress()

```

Comment:
Minor, but could we have this test directly after the other RemoveDelegateFromCourse test.